tutorial/transport/linopy_model.py

```diff
@@ -0,0 +1,126 @@
+# m = linopy.Model()
+# i = {"Canning Plants": ["seattle", "san-diego"]}
+# j = {"Markets": ["new-york", "chicago", "topeka"]}
+# a = xr.DataArray([350, 600], coords=i, name="capacity of plant i in cases")
+# b = xr.DataArray([325, 300, 275], coords=j, name="demand at market j in cases")
+# d = xr.DataArray(
+#     [[2.5, 1.7, 1.8], [2.5, 1.8, 1.4]],
+#     coords=i | j,
+#     name="distance in thousands of miles",
+# )
+# f = 90  # Freight in dollars per case per thousand miles
+# c = d * f / 1000
+# c.name = "transport cost in thousands of dollars per case"
+# x = m.add_variables(lower=0.0, coords=c.coords, name="Shipment quantities in cases")
+# con = x.sum(dim="Markets") <= a
+# con1 = m.add_constraints(con, name="Observe supply limit at plant i")
+# con = x.sum(dim="Canning Plants") >= b
+# con2 = m.add_constraints(con, name="Satisfy demand at market j")
+# obj = c * x
+# m.add_objective(obj)
+# print(linopy.available_solvers)
+# m.solve("highs")
+# print(x.solution)
+
+import linopy
+import pandas as pd
+
+from ixmp4.core import Equation, IndexSet, Parameter, Scalar
+from ixmp4.core import OptimizationVariable as Variable
+
+
+def create_set(indexset: IndexSet, name: str | None = None) -> pd.Index:
+    return pd.Index(indexset.elements, name=name or indexset.name)
+
+
+def create_parameter(
+    parameter: Parameter, index: pd.Index | list[pd.Index], name: str
+) -> pd.Series:
+    if isinstance(index, list):
+        index = pd.MultiIndex.from_product(index)
+
+    return pd.Series(data=parameter.values, index=index, name=name)
+
+
+def create_dantzig_model(
+    i: IndexSet,
+    j: IndexSet,
+    a: Parameter,
+    b: Parameter,
+    d: Parameter,
+    f: Scalar,
+) -> linopy.Model:
+    m = linopy.Model()
+    i_set = create_set(indexset=i, name="Canning Plants")
```

Comment:
I guess that you don't need the `create_set()` utility.... And I'm not sure why you give long names instead of the short ones used in the mathematical notation (why "Canning Plants", not "i").
```suggestion
    i_set = pd.Index(
       run.optimization.indexsets.get("i").elements, 
       name="i",
    )
```

Reply:
The pyomo and linopy tutorials use long names and I'm usually a fan of that as well. Because on the flipside: why would you use `i` instead of `Canning Plants`? Maybe you can save a little bit of time, but you create more confusion because people new to the problem then constantly have to check: what was `i` again? And we could even do something like 
```python
i_name = "Canning Plants"
i_set = create_set(indexset=i, name=i_name)
```
so that we have to completely type it only once and still have clarity in the variables and results: what did this actually mean?

Reply:
Huh, I was just tinkering with this, but now I'm thinking: if this is supposed to be a tutorial, the way we have it now might do that best: showing people how to work with linopy and ixmp4. Because it might not be clear, that the dimension names of the linopy objects are allowed to differ or where exactly these differences can occur. With the current setup, we would demonstrate that.
So even without harmonizing everything to read either `i` or `canning_plants`, I think the current setup is useful.

BTW: I have removed the helper function as suggested here.

Reply:
There is no right or wrong between short names (close to the math) or long names (readability), either in the notebook or the model-code, as long as it’s understandable and only the run is passed as argument, not the individual items.